### PR TITLE
Add clockevent check case

### PR DESCRIPTION
**Pull request by HuijingHei**
Not support on rhel6

### CODE REVIEW

File: WS2012R2/lisa/remote-scripts/ica/check_clockevent.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    fi
+
+}
+
+#
+# MAIN SCRIPT
+#
+GetDistro
+case $DISTRO in
+    redhat_6)
```
Comment (chvalean) [inline WS2012R2/lisa/remote-scripts/ica/check_clockevent.sh]: it should also contain centos_6

File: WS2012R2/lisa/remote-scripts/ica/check_clockevent.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        LogMsg "WARNING: $DISTRO does not support clockevent."
+        UpdateSummary "WARNING: $DISTRO does not support clockevent."
+        SetTestStateSkipped
+        exit 1
+        ;;
+    redhat_7 | redhat_8)
```
Comment (chvalean) [inline WS2012R2/lisa/remote-scripts/ica/check_clockevent.sh]: please also add centos_7 and centos_8

File: WS2012R2/lisa/remote-scripts/ica/check_clockevent.sh [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        CheckClockEvent
+        ;;
+    fedora_* )
+        CheckClockEvent
+        ;;
+    *)
```
Comment (chvalean) [inline WS2012R2/lisa/remote-scripts/ica/check_clockevent.sh]: we usually also support ubuntu, if it has this feature then we should handle it

File: WS2012R2/lisa/xml/CoreTests.xml [hunk 4]
```diff
@@ -382,6 +383,19 @@
             <noReboot>True</noReboot>
         </test>
 
+        <test>
+            <testName>check_clockevent</testName>
+            <testScript>check_clockevent.sh</testScript>
+            <files>remote-scripts/ica/check_clockevent.sh</files>
```
Comment (chvalean) [inline WS2012R2/lisa/xml/CoreTests.xml]: multiple files can be handled in a single definition by using coma

**Comment by HuijingHei** [comment]
Hi Chris, I have updated scripts according to your comments. Thanks!

**Pull request merged**